conans/client/generators/visualstudio.py

```diff
@@ -3,6 +3,7 @@
 
 from conans.model import Generator
 from conans.paths import BUILD_INFO_VISUAL_STUDIO
+from conans.client.tools.files import _valid_lib_extensions
```

Comment:
Not nice to have to import a private thing from another module, better make it public

Reply:
done!